src/compiler/index.ts

```diff
@@ -22,3 +22,4 @@ export { path } from './sys/modules/path';
 export { transpile, transpileSync } from './transpile';
 export { version, versions, vermoji, buildId } from '../version';
 export { ts };
+export { InMemoryFileSystem, FsWriteResults } from './sys/in-memory-fs';
```

Comment:
Should we be exporting `InMemoryFileSystem`? Previously that was a private type. It's not that we can't export it, but it does expand the scope of things that are public facing that we maintain

Reply:
yeah it's not necessary I think, tbh I don't remember why I added it here so I'll remove it